core-developers/memorialization.rst

```diff
@@ -0,0 +1,134 @@
+* The user's trust level is reset to ``1: basic user`` (trust level 0 doesn't allow links in "About Me");
+* The user's "associated accounts" (like GitHub) that provide an alternative
+  login method, are all disconnected;
+* The user's API keys are revoked;
+* The user's admin or moderator right is revoked;
+* The user's primary email address is reset to ``USERNAME@in-memoriam.invalid`` and
+  secondary email addresses are removed (this step requires the administrator
+  to contact Discourse.org staff via ``team@discourse.org``)
+
+The "in memoriam" Discourse topic mentioned above is best created by
+a community member close to the deceased.
+
+The general best practice for deceased community members on
+Discourse-powered forums is described `here <https://meta.discourse.org/t/best-practices-for-deceased-community-members/146210>`_.
+
+python.org email account
+------------------------
+
+The PSF staff member emails ``postmaster@python.org`` to ask the email
+administrator to:
+
+* remove SMTP access from ``USERNAME@python.org``;
+* reset the password to POP3/IMAP for ``USERNAME@python.org``;
+* disable email forwarding, if set up, for ``USERNAME@python.org``;
+* remove this email from all mailing lists under ``@python.org``;
+* remove any known alternate emails for the same user from all mailing lists
+  under ``@python.org``.
+
+python.org admin
+----------------
+
+* The user's account (``/admin/users/user``) is deactivated (NOT deleted)
+  and their staff and superuser status is unchecked;
+* The user's password is reset to a long random string;
+* The user's primary email address is set to ``USERNAME@in-memoriam.invalid``
+  and set as unverified;
+* The user's secondary email addresses are deleted;
+* The user's API keys (both on the account and ``tastypie``) are deleted;
+* The user's "I would like to be a PSF Voting Member" field is cleared.
+
+devguide.python.org
+-------------------
+
+* The user is marked as deceased in `developers.csv <https://github.com/python/devguide/blob/main/core-developers/developers.csv>`_;
+* The user is removed from the `Experts Index <https://github.com/python/devguide/blob/main/core-developers/experts.rst>`_.
+
+bugs.python.org
+---------------
+
+While the issue tracker was migrated to GitHub, the Roundup instance
+is still up for historical purposes.
+
+* the PSF staff member logs into ``bugs.nyc1.psf.io``;
+* the PSF staff member runs ``roundup-admin`` to set the user's email
+  address to ``USERNAME@in-memoriam.invalid``;
+* the user's alternate emails are removed;
+* the user's password is reset to a long random string;
+* the PSF staff member removes any active login sessions from Postgres.
+
+SSH server access
+-----------------
+
+* The user is removed from Salt configuration for the PSF infrastructure
+  in `/pillar/base/users <https://github.com/python/psf-salt/tree/main/pillar/base/users>`_.
+
+PyPI
+----
+
+* The PSF staff member notifies PyPI admins to mark the user as inactive,
```

Comment:
Can mention `admin@pypi.org` email address for this use-case.

Reply:
Good point. Made the change. Thanks for the review!